src/cuda/cuda-command.cpp

```
@@ -665,8 +828,12 @@ void CommandExecutor::cmdInsertDebugMarker(const commands::InsertDebugMarker& cm
 
 void CommandExecutor::cmdWriteTimestamp(const commands::WriteTimestamp& cmd)
 {
```

Cppcheck (2.20.0)

[style] The function 'getTextureDimension' is never used.

(unusedFunction)